frontend/src/metabase/components/LastEditInfoLabel/LastEditInfoLabel.tsx

```diff
@@ -93,14 +93,20 @@ function LastEditInfoLabel({
 
   return (
     <Tooltip disabled={!timeLabel} {...tooltipProps}>
-      <TextButton
-        size="small"
-        className={className}
-        onClick={onClick}
-        data-testid="revision-history-button"
-      >
-        {children}
-      </TextButton>
+      {onClick ? (
+        <TextButton
+          size="small"
+          className={className}
+          onClick={onClick}
+          data-testid="revision-history-button"
+        >
+          {children}
+        </TextButton>
+      ) : (
+        <Text size="sm" fw="bold" c="var(--mb-color-text-secondary)">
```

Comment:
Here's how the new text looks compared to the previous one. It's slightly smaller which is negligible.
![Screenshot 2024-10-11 at 11 23 25 PM](https://github.com/user-attachments/assets/22d1707e-04e1-4067-b86f-ed8ba25785c0)